app/src/pages/index.tsx

```diff
@@ -55,9 +55,20 @@ export default function Home({ capabilityStatement }: InferGetServerSidePropsTyp
         <Anchor href="https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html">
```

Comment:
Thoughts on updating the link to point to the HL7 site (http://hl7.org/fhir/us/cqfmeasures/measure-repository-service.html) rather than build.fhir.org?